plotting/plotter.py

```diff
@@ -321,6 +323,77 @@ def odv_ascii(self, cruise="", variables=[], variable_units=[],
                     buf.write("\n")
 
             return (buf.getvalue(), self.mime, self.filename)
+        
+        # Defining igoss_ascii Helper Functions:
+    def format_igoss_header(self, points, time):
+        igoss_header = np.array([],dtype=str)
+        for p in points:        
+            latitude = p[0]
+            longitude = p[1]
+            if latitude > 0. and longitude > 0. :
+                quad_id = 1
+            elif latitude < 0. and longitude > 0. :
+                quad_id = 3
+            elif latitude < 0. and longitude < 0. :
+                quad_id = 5
+            elif latitude > 0. and longitude < 0. :
+                quad_id = 7
```

Comment:
What are these quad_id's, and what does the 1, 3, 5, 7 represent?

Reply:
This format doesn't allow us to use negative coordinates or cardinal directions for the lat and lon values. Instead, it divides the globe into four quadrants:
 1 North East
 3 South East
 5 South West
 7 North West
The operator and software ingesting this data can use this number to determine where the coordinates are.

Reply:
![image](https://user-images.githubusercontent.com/5572045/155157118-210e0b81-5493-41c0-a9c0-8d49e5e20c43.png)

Reply:
Yup haha.